backend/src/main/java/com/htecgroup/skynest/SkyNestApplication.java

```diff
@@ -24,11 +25,15 @@ public WebMvcConfigurer corsConfigurer() {
     return new WebMvcConfigurer() {
       @Override
       public void addCorsMappings(CorsRegistry registry) {
+        String publicIp = UrlUtil.SKYNEST_HOST;
+        String frontendPort = String.valueOf(UrlUtil.FRONTEND_PORT);
         registry
             .addMapping("/**")
             .allowedMethods("*")
-            .allowedOrigins("http://localhost:3000")
-            .allowedOriginPatterns("http://localhost:3000/*")
+            .allowedOrigins(
+                "http://localhost:3000", String.format("http://%s:%s", publicIp, frontendPort))
```

Comment:
so you already have that values now in separate variables right? Why don't you use them instead of the hard coded ones.

Reply:
It's hard-coded so that so that FE could send requests from their local builds. These requests would have <code>Origin: http://localhost:3000</code>.

Reply:
Again this way you are hardcoding into the app, not based on environment where it is loaded.
The point is this to be restricted based on specific env.
In this case if you add skynest.host=http://localhost in the application-local.properties, and if FE is starting the app locally with corresponding profile - local, as they should, it will work right? 

On the other hand, when you deploy this to the VM it will work with value set in ${SKYNEST_HOST}, so no changes will be done in the code, right?

Reply:
Yeah, it's probably simpler if FE just tests their local build with our local build. The deployed versions should only work with each other anyway.